lib/sandboxed_renderer/init.js

```diff
@@ -43,15 +51,19 @@ preloadProcess.hang = () => binding.hang()
 preloadProcess.getProcessMemoryInfo = () => binding.getProcessMemoryInfo()
 preloadProcess.getSystemMemoryInfo = () => binding.getSystemMemoryInfo()
 preloadProcess.argv = binding.getArgv()
-process.platform = preloadProcess.platform = electron.remote.process.platform
-process.execPath = preloadProcess.execPath = electron.remote.process.execPath
-process.env = preloadProcess.env = electron.remote.process.env
+preloadProcess.platform = process.platform = platform
+preloadProcess.execPath = process.execPath = execPath
+preloadProcess.env = process.env = env
+
 process.on('exit', () => preloadProcess.emit('exit'))
 
 // This is the `require` function that will be visible to the preload script
 function preloadRequire (module) {
-  if (preloadModules.has(module)) {
-    return preloadModules.get(module)
+  if (loadedModules.has(module)) {
+    return loadedModules.get(module)
+  }
+  if (remoteModules.has(module)) {
+    return require(module)
```

Comment:
shouldn't this be electron.remote.require instead?

Reply:
No, we still require these files to emulate a real node.js require for electron renderer modules that are reused in sandbox. Those are bundled in the executable and don't know they are running in a browserify environment.

Reply:
What I mean is, electron modules such as crashReporter won't use this `require` function, so we need to tell browserify that when crashReporter requires `fs`, it is requiring `/lib/sandboxed_renderer/api/exports/fs.js` and not browserify shim.

Reply:
ok